bittide-instances/README.md

```diff
@@ -64,3 +64,14 @@ Example:
 ```
 cabal run -- bittide-instances:shake scatterUnitWb:netlist
 ```
+
+## Hardware
+Prerequisites to running shake:
+- Source VivadoEnterprise
+- The environment variable `SYNTHESIS_PART` is `xcku040-ffva1156-2-e`
```

Comment:
This isn't strictly necessary, but it's helpful to point out that you can change the default. Could you move that to the intro of this README?